<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="signup" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>40</y>
            <width>221</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 19pt &quot;Algerian&quot;;</string>
          </property>
          <property name="text">
           <string>CREATE ACCOUNT</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>140</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;.VnArial&quot;;
background-color: rgba(0, 0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>180</y>
            <width>321</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter your email</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>220</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;.VnArial&quot;;
background-color: rgba(0, 0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>250</y>
            <width>321</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter your password</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>300</y>
            <width>371</width>
            <height>17</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;.VnArial&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>I have read and agree to The Cafe's Privacy Policy</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>320</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-radius: 5px;</string>
          </property>
          <property name="text">
           <string>Sign up</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_1">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>370</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 255);</string>
          </property>
          <property name="text">
           <string>Already have an account?</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>100</y>
            <width>321</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter your full name</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>-10</y>
            <width>181</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../../PTI/Screenshot 2024-03-03 at 08.17.08.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../image/image.qrc"/>
 </resources>
 <connections/>
</ui>
